communityMUD/commands/unloggedin.py

```diff
@@ -0,0 +1,138 @@
+    Terms of Service and Code of Conduct during the creation process.
+
+    If you have spaces in your name, enclose it in double quotes.
+    """
+
+    key = "create"
+    aliases = ["cre", "cr"]
+    locks = "cmd:all()"
+    arg_regex = r"\s.*?|$"
+
+    def func(self):
+        """Do checks and create account, with ToS and CoC confirmation"""
+
+        session = self.caller
+        args = self.args.strip()
+
+        address = session.address
+
+        # Get account class
+        Account = class_from_module(settings.BASE_ACCOUNT_TYPECLASS)
+
+        # extract double quoted parts
+        parts = [part.strip() for part in re.split(r"\"", args) if part.strip()]
+        if len(parts) == 1:
+            # this was (hopefully) due to no quotes being found
+            parts = parts[0].split(None, 1)
+        if len(parts) != 2:
+            string = (
+                "\n Usage (without <>): create <n> <password>"
+                "\nIf <n> or <password> contains spaces, enclose it in double quotes."
+            )
+            session.msg(string)
+            return
+
+        username, password = parts
+
+        # pre-normalize username so the user know what they get
+        non_normalized_username = username
+        username = Account.normalize_username(username)
+        if non_normalized_username != username:
+            session.msg(
+                "Note: your username was normalized to strip spaces and remove characters "
+                "that could be visually confusing."
+            )
+
+        # have the user verify their new account was what they intended
+        answer = yield (
+            f"You want to create an account '{username}' with password '{password}'."
+            "\nIs this what you intended? [Y]/N?"
+        )
+        if answer.lower() in ("n", "no"):
+            session.msg("Aborted. If your user name contains spaces, surround it by quotes.")
+            return
+
+        # Present Terms of Service and Code of Conduct
+        tos_text = """
+|y==============================================================|n
+|rTERMS OF SERVICE|n
+
+By creating an account, you agree to:
+
+1. Respect all users and staff
+2. Follow all game rules and policies
+3. Not exploit bugs or game mechanics
+4. Not engage in harassment or abuse
+5. Allow your account data to be stored
+
+This is a placeholder for the full Terms of Service, which will be 
+available later.
+
+|y==============================================================|n
+"""
+        coc_text = """
+|y==============================================================|n
+|rCODE OF CONDUCT|n
+
+As a player, you will:
+
+1. Be respectful of other players
+2. Use appropriate language
+3. Not engage in discrimination or harassment
+4. Respect the privacy of other players
+5. Follow staff instructions
+
+This is a placeholder for the full Code of Conduct, which will be 
+available later.
+
+|y==============================================================|n
+"""
```

Comment:
nitpick but I'd put these somewhere else since they are static and won't be changing. 
We could go as far as put them in a file and read them, but just the top of this file would suffice, no need to clutter up the code here.